fontforge/glyphcomp.c

```diff
@@ -197,8 +197,8 @@ return( true );
 			 ((dy=s->to->me.y-here->y)<=3 || dy<=3*err ) && (dy>=-3 || dy>=-3*err)) ) {
 		    s = s->to->next;
 		    t = 0;
-		    if ( (adx = (3*s->splines[0].a* t + 2*s->splines[0].b)* t + s->splines[0].c)<0 ) adx = -adx;
-		    if ( (ady = (3*s->splines[1].a* t + 2*s->splines[1].b)* t + s->splines[1].c)<0 ) ady = -ady;
+		    if ( s!=NULL && (adx = (3*s->splines[0].a* t + 2*s->splines[0].b)* t + s->splines[0].c)<0 ) adx = -adx;
```

Comment:
I think this code may have been assuming that only closed contours would be compared. In any case s->to->next can wind up being NULL in practice and this prevents a segfault.